### DISCUSSION TITLE
Deployment plan issue HttpClient Timeout 100 seconds

### DISCUSSION THREAD

**Discussion by DannyT** [OrchardCMS/OrchardCore#13469] [Q&A] [answered]
I'm trying to deploy a large amount of content items via a deployment plan to a remote instance and hitting the HttpClient timeout. Is this configurable anywhere by any chance?

`The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.`

**Comment by hishamco** [comment, marked as answer]
I think there's an issue that describing a solution by divide the recipe into chunks of recipes to avoid such problem

**Comment by Piedone** [comment]
I'm afraid the only way would be to copy [`ExportRemoteInstanceController`](https://github.com/OrchardCMS/OrchardCore/blob/5eca92d48392891f4db29cb047984925bcc5f360/src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ExportRemoteInstanceController.cs#L21) to your module, set the timeout, and use it to override the built-in controller.

That being said, even a 100s HTTP request is really stretching it, so yeah, like Hisham I'd rather recommend splitting the recipes into multiple ones.

**Comment by hishamco** [comment]
Also, I've been thinking for a while about achieving this using a job queue, but with the current implementation may be splitting them into chunks is a good option that you can go with it

**Comment by DannyT** [comment]
Thanks all, I managed to use a Queried Content Step with paging to do it as you suggested 🙌
